pkg/replication/replication.go

```diff
@@ -195,6 +195,10 @@ func (c *Config) AddRule(opts Options) error {
 		},
 		DeleteMarkerReplication: DeleteMarkerReplication{Status: dmStatus},
 		DeleteReplication:       DeleteReplication{Status: vDeleteStatus},
+		SourceSelectionCriteria: SourceSelectionCriteria{
+			ReplicaModifications: ReplicaModifications{
+				Status: Enabled},
```

Comment:
```suggestion
				Status: Enabled,
				},
```

Reply:
BTW should we do this by default?

Reply:
If we do not enable this by default, we will need additional flags in mc to support toggling this status. It seems to make sense for bi-directional replication to keep source and replica in sync

Reply:
Hmm good - add a comment on why we enforce it @poornas - and also why it makes sense by referencing the link from AWS - such that our choice is to make it default

Reply:
done